### PR TITLE
Stylecheck

### PR DESCRIPTION
**Pull request by ChristinaXu2017**
# Description

I modified java code followed by the google style check instruction, mostly in space, line break, bracket issues. Here google_check.xml is updated, it was "^[a-z][a-z0-9][a-zA-Z0-9]*$" but now it is "^[a-z][a-zA-Z0-9]*$".  now variable like "eId", "pLatch" can be accept.  At moemnt all changes is  only made inside qprofiler2. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

same unit test 

# Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [ X] I have made corresponding changes to the documentation
- [X ] My changes generate no new warnings
- [X ] I have added tests that prove my fix is effective or that my feature works
- [X ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 1]
```diff
@@ -67,43 +67,46 @@
...
+		
+		//  no of threads - Producer
 		Object threadNumberProducer = options.valueOf("ntProducer"); 
-		if (null != threadNumberProducer)
+		if (null != threadNumberProducer) {
 			noOfProducerThreads =  (Integer) threadNumberProducer;
-		
-		// maxRecords
+		}
+		 //  maxRecords
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:81]: same as above. This looks like the result of a global find/replace? But adding space before all the inline comments makes many of them not align with the code below

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 2]
```diff
@@ -67,43 +67,46 @@
...
 		Object maxRecordsObject = options.valueOf("maxRecords"); 
-		if (null != maxRecordsObject)
-			maxRecords =  (Integer) maxRecordsObject;		
+		if (null != maxRecordsObject) {
+			maxRecords =  (Integer) maxRecordsObject;	
+		}	
 		
-		// log file
+		 //  log file
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:87]: same

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 3]
```diff
@@ -67,43 +67,46 @@
...
 		log = (String) options.valueOf("log");
-		// logLevel
+		 //  logLevel
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:89]: same

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 4]
```diff
@@ -67,43 +67,46 @@
...
 		logLevel = (String) options.valueOf("loglevel");
 		
-		// vcf mode: format field name
+		 //  vcf mode: format field name
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:92]: same

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 5]
```diff
@@ -67,43 +67,46 @@
...
 		List<String> formatArgs = (List<String>) options.valuesOf("format");
 		formats = new String[formatArgs.size()];
 		formatArgs.toArray(formats);
 				
-		// inputs
+		 //  inputs
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:97]: same

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 6]
```diff
@@ -67,43 +67,46 @@
...
 		List<String> inputs = (List<String>) options.valuesOf("input");
 		fileNames = new String[inputs.size()];
 		inputs.toArray(fileNames);
 
-		// indexes
+		 //  indexes
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:102]: same

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 7]
```diff
@@ -67,43 +67,46 @@
...
 		List<String> indexes = (List<String>) options.valuesOf("index");
 		indexFileNames = new String[indexes.size()];
 		indexes.toArray(indexFileNames);
 		
-		// output
+		 //  output
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:107]: same

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 8]
```diff
@@ -67,43 +67,46 @@
...
 		outputFileName = (String) options.valueOf("output");		
-		if ( ! options.nonOptionArguments().isEmpty()) {
+		if ( ! options.nonOptionArguments().isEmpty()) {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:109]: an extra whitespace has been introduced here.

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 9]
```diff
@@ -137,8 +140,8 @@ boolean hasLogLevelOption() {
 		return indexFileNames; 
 	}
 		
-	//vcf mode
-	String[] getFormats(){ 
+	 // vcf mode
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:143]: same

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 10]
```diff
@@ -163,17 +166,23 @@ String getLogLevel() {
...
+		return outputFileName; 
+	}
+	
+	void displayHelp() throws Exception { 	
+		parser.printHelpOn(System.err);
+	}
 	
-	String getValidation() {
-		if (options.has("validation")) {
+	String getValidation() {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Options2.java:181]: extra whitespace after the open { here and the line below

File: qprofiler2/src/org/qcmg/qprofiler2/QProfiler2.java [hunk 11]
```diff
@@ -70,10 +71,10 @@
 	 * method is invoked so we can assume that the class fields are populated
 	 * and ready for us to use.
 	 */
-	protected int engage() throws Exception {
+	protected int engage() throws Exception { 
 		Element root = XmlElementUtils.createRootElement( "qProfiler", null);
 		
-		// Create new Summary object ready to hold our processing
+		 //  Create new Summary object ready to hold our processing
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/QProfiler2.java:77]: same in this file - a whitespace has been added before and after _every_ occurrence of `//` where for most of them it should only have been added after.

File: qprofiler2/src/org/qcmg/qprofiler2/QProfilerSummary2.java [hunk 12]
```diff
@@ -24,7 +24,7 @@
 
 import org.w3c.dom.Element;
 
-public class QProfilerSummary2 {
+public class QProfilerSummary2 {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/QProfilerSummary2.java:27]: same in this file - not sure why space has been added after all the opening block parentheses `{` .

File: qprofiler2/src/org/qcmg/qprofiler2/Summarizer.java [hunk 13]
```diff
@@ -8,7 +8,7 @@
  */
 package org.qcmg.qprofiler2;
 
-public interface Summarizer {
+public interface Summarizer {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/Summarizer.java:11]: same in this file (maybe all files?) - why is there a whitespace after all the opening block parentheses?

File: qprofiler2/src/org/qcmg/qprofiler2/SummaryReport.java [hunk 14]
```diff
@@ -20,7 +21,7 @@
 import org.qcmg.common.util.FileUtils;
 import org.w3c.dom.Element;
 
-public abstract class SummaryReport {
+public abstract class SummaryReport {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/SummaryReport.java:24]: why are we adding whitespace after open parenthesis of block

File: qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummarizer2.java [hunk 15]
```diff
@@ -33,31 +33,31 @@
...
 	
 	private int maxRecords;
 	private String validation;
 	private boolean isFullBamHeader;
 
-	private final static QLogger logger = QLoggerFactory.getLogger(BamSummarizer2.class);	
-	public BamSummarizer2() {}	// default constructor	
-	public BamSummarizer2( int maxRecords, String validation, boolean isFullBamHeader) {
+	private static final QLogger logger = QLoggerFactory.getLogger(BamSummarizer2.class);	
+	public BamSummarizer2() { }	 //  default constructor
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummarizer2.java:44]: Empty blocks shouldn't contain space 
https://google.github.io/styleguide/javaguide.html#s4.1.3-braces-empty-blocks

File: qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummarizerMT2.java [hunk 16]
```diff
@@ -38,21 +33,24 @@
...
 	private final int maxRecords;
 	private final String validation;
 	private final boolean isFullBamHeader;
 	ValidationStringency vs;
-	private final static String UNMAPPED_READS = "Unmapped";
-
+	private static final String UNMAPPED_READS = "Unmapped";
 	
-	public BamSummarizerMT2(int noOfProducerThreads, int noOfThreads, int maxNoOfRecords, String validation,boolean isFullBamHeader) {
+	public BamSummarizerMT2(int noOfProducerThreads, int noOfThreads, int maxNoOfRecords, String validation,boolean isFullBamHeader) {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummarizerMT2.java:53]: Same in this file - whitespace after open parentheses

File: qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummarizerMT2.java [hunk 17]
```diff
@@ -62,121 +60,127 @@ public BamSummarizerMT2(int noOfProducerThreads, int noOfThreads, int maxNoOfRec
 	}
 		
 	@Override
-	public SummaryReport summarize(String input, String index) throws Exception {
+	public SummaryReport summarize(String input, String index) throws Exception {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummarizerMT2.java:63]: Lots of spaces after open parentheses

File: qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummaryReport2.java [hunk 18]
```diff
@@ -43,75 +42,81 @@
...
-import org.qcmg.qprofiler2.summarise.ReadIDSummary;
+import org.qcmg.qprofiler2.summarise.ReadIdSummary;
 import org.qcmg.qprofiler2.util.CycleSummaryUtils;
 import org.qcmg.qprofiler2.util.FlagUtil;
 import org.qcmg.qprofiler2.util.SummaryReportUtils;
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
-public class BamSummaryReport2 extends SummaryReport {	
+public class BamSummaryReport2 extends SummaryReport {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummaryReport2.java:52]: same

File: qprofiler2/src/org/qcmg/qprofiler2/bam/TagSummaryReport2.java [hunk 19]
```diff
@@ -21,86 +21,98 @@
 import htsjdk.samtools.SAMRecord.SAMTagAndValue;
 import htsjdk.samtools.SAMTagUtil;
 
-//for sam record tag information
-public class TagSummaryReport2 {
+ // for sam record tag information
+public class TagSummaryReport2 {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/bam/TagSummaryReport2.java:25]: same

File: qprofiler2/src/org/qcmg/qprofiler2/cohort/CohortSummarizer.java [hunk 20]
```diff
@@ -21,14 +21,14 @@
 import org.w3c.dom.NodeList;
 
 
-public class CohortSummarizer implements Summarizer {
-	private final static QLogger logger = QLoggerFactory.getLogger(CohortSummarizer.class);
+public class CohortSummarizer implements Summarizer {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/cohort/CohortSummarizer.java:24]: same

File: qprofiler2/src/org/qcmg/qprofiler2/cohort/CohortSummaryReport.java [hunk 21]
```diff
@@ -15,41 +15,48 @@
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
-public class CohortSummaryReport extends SummaryReport {
+public class CohortSummaryReport extends SummaryReport {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/cohort/CohortSummaryReport.java:18]: same

File: qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummarizer.java [hunk 22]
```diff
@@ -30,31 +30,31 @@
 import org.qcmg.qprofiler2.Summarizer;
 import org.qcmg.qprofiler2.SummaryReport;
 
-public class FastqSummarizer implements Summarizer {
+public class FastqSummarizer implements Summarizer {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummarizer.java:33]: same

File: qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummarizerMT.java [hunk 23]
```diff
@@ -28,19 +29,19 @@
 import org.qcmg.qprofiler2.Summarizer;
 import org.qcmg.qprofiler2.SummaryReport;
 
-public class FastqSummarizerMT implements Summarizer {
+public class FastqSummarizerMT implements Summarizer {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummarizerMT.java:32]: same

File: qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java [hunk 24]
```diff
@@ -22,44 +23,49 @@
...
-import org.qcmg.qprofiler2.summarise.ReadIDSummary;
+import org.qcmg.qprofiler2.summarise.ReadIdSummary;
 import org.qcmg.qprofiler2.util.SummaryReportUtils;
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
-public class FastqSummaryReport extends SummaryReport {
-	public final static String badBaseComment = " the number of reads containing a given number of bad bases (. or N) ";	
-	public final static String badQualComment = "the number of reads containing a given number of bases with bad quality scores (<10)";
+public class FastqSummaryReport extends SummaryReport {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java:31]: same

File: qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java [hunk 25]
```diff
@@ -22,44 +23,49 @@
...
+	public static final String badBaseComment = " the number of reads containing a given number of bad bases (. or N) ";	
+	public static final String badQualComment = "the number of reads containing a given number of bases with bad quality scores (<10)";
 	
 	private static final Character c = Character.MAX_VALUE;
 	private static final Integer i = Integer.MAX_VALUE;	
 	private static final int ReportErrNumber = 100;
 	AtomicLong errNumber = new AtomicLong();	
 	
-	//SEQ
+	 // SEQ
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java:40]: same

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java [hunk 26]
```diff
@@ -29,13 +31,15 @@
 
 /**
  * Class that tallies by cycle using java generics 
+ *
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java:34]: line below is unaligned by one space

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java [hunk 27]
```diff
@@ -111,20 +115,24 @@ public int getArrayPosition(int cycle, int key) {
 	 * @param position of the array
 	 * @return a pair of cycle number and value which is stored in int. eg. array position 100, stores cycle 20 with 'A' (65)
 	 */
-	public int[] getCycleKeyFromArrayPosition(int position) {
+	public int[] getCycleKeyFromArrayPosition(int position) { 
 		int key = ( position >>> cycleMask.get());
 		int cycle  = position & (( 1 << cycleMask.get()) - 1);
-		return new int[] {cycle,key};
+		return new int[] { cycle,key};
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java:121]: space

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java [hunk 28]
```diff
@@ -20,124 +20,131 @@
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
-public class KmersSummary {
+public class KmersSummary {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java:23]: same

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java [hunk 29]
```diff
@@ -20,124 +20,131 @@
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
-public class KmersSummary {
+public class KmersSummary { 
 		
-	//init 0, eventially store the biggest cycle number
+	 // init 0, eventially store the biggest cycle number
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java:25]: same

File: config/google_check.xml [hunk 30]
```diff
@@ -122,12 +122,12 @@
...
         </module>
         <module name="MemberName">
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$"/>
+            <property name="format" value="^[a-z][a-zA-Z0-9]*$"/>
             <message key="name.invalidPattern"
              value="Member name ''{0}'' must match pattern ''{1}''."/>
         </module>
         <module name="ParameterName">
-            <property name="format" value="^[a-z]([a-z0-9][a-zA-Z0-9]*)?$"/>
+            <property name="format" value="^^[a-z][a-zA-Z0-9]*$"/>
```
[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply config/google_check.xml]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 31]
```diff
@@ -32,9 +32,9 @@
 	private static final String VALIDATION_STRINGENCY_OPTION_DESCRIPTION = Messages.getMessage(msgResource, "VALIDATION_STRINGENCY_DESCRIPTION");
 	private static final String FULL_BAMHEADER_OPTION_DESCRIPTION = Messages.getMessage(msgResource, "FULL_BAMHEADER_OPTION_DESCRIPTION");
 	
-	//vcf mode
+	 // vcf mode
```
[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:35]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java [hunk 32]
```diff
@@ -67,43 +67,46 @@
 		parser.posixlyCorrect(true);
 		options = parser.parse(args);
 		
-		// no of threads - Consumer
+		 //  no of threads - Consumer
```
[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:70]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 1 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:81]: done

File: qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java (hunk 25 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java:40]: done

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java (hunk 26 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java:34]: done

File: config/google_check.xml (hunk 30 above)
[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply config/google_check.xml]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 31 above)
[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:35]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 32 above)
[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:70]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 1 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:81]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 2 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:87]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 3 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:89]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 4 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:92]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 5 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:97]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 6 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:102]: done

File: qprofiler2/src/org/qcmg/qprofiler2/Options2.java (hunk 7 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/Options2.java:107]: done

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java (hunk 27 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/summarise/CycleSummary.java:121]: done

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java (hunk 28 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java:23]: done

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java (hunk 29 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/summarise/KmersSummary.java:25]: done

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/SampleSummary.java [hunk 33]
```diff
@@ -23,158 +23,167 @@
...
+		}
 		
 		int commaIndex = ad.indexOf(Constants.COMMA);  
 		int vaf = 0;
 		
 		/*
 		 *  vaf needs to equal the sum of all the numbers in the AD field, apart from the first number (which is the reference count)
 		 */
-		while (commaIndex > -1) {
+		while (commaIndex > -1) {
```
Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/summarise/SampleSummary.java:72]: On many lines this file has the trailing space after open `{` block

Comment (delocalizer) [inline qprofiler2/src/org/qcmg/qprofiler2/summarise/SampleSummary.java:72]: In several places this file has got a new trailing space after open `{` blocks that shouldn't be there

File: qprofiler2/test/org/qcmg/qprofiler2/OptionsTest.java [hunk 34]
```diff
@@ -4,32 +4,32 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 import org.junit.Test;
-import org.qcmg.qprofiler2.Options2;
+import org.qcmg.qprofiler2.Options;
 
-public class OptionsTest {
+public class OptionsTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/OptionsTest.java:9]: In several places in this file there is now a new trailing space after open `{` blocks that shouldn't be there.

File: qprofiler2/test/org/qcmg/qprofiler2/QProfiler2Test.java [hunk 35]
```diff
@@ -24,7 +24,7 @@
 import org.qcmg.common.messages.Messages;
 import org.qcmg.qprofiler2.QProfiler2;
 
-public class QProfiler2Test {
+public class QProfiler2Test {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/QProfiler2Test.java:27]: Same in this file.

File: qprofiler2/test/org/qcmg/qprofiler2/bam/BamSummarizerTest.java [hunk 36]
```diff
@@ -18,10 +18,10 @@
 import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
-import org.qcmg.qprofiler2.bam.BamSummarizer2;
-import org.qcmg.qprofiler2.bam.BamSummaryReport2;
+import org.qcmg.qprofiler2.bam.BamSummarizer;
+import org.qcmg.qprofiler2.bam.BamSummaryReport;
 
-public class BamSummarizerTest {
+public class BamSummarizerTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/bam/BamSummarizerTest.java:24]: same

File: qprofiler2/test/org/qcmg/qprofiler2/bam/BamSummaryReportMetricsTest.java [hunk 37]
```diff
@@ -34,23 +34,23 @@
  * check <bamMetrics>...<sequenceMetrics name="..." readCount="...">.
  *
  */
-public class BamSummaryReportMetricsTest {
+public class BamSummaryReportMetricsTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/bam/BamSummaryReportMetricsTest.java:37]: same

File: qprofiler2/test/org/qcmg/qprofiler2/bam/BamSummaryReportTest.java [hunk 38]
```diff
@@ -16,22 +16,22 @@
...
+import org.qcmg.qprofiler2.bam.BamSummaryReport;
 import org.qcmg.qprofiler2.summarise.CycleSummaryTest;
 import org.qcmg.qprofiler2.summarise.PairSummaryTest;
 import org.qcmg.qprofiler2.summarise.PositionSummary;
 import org.qcmg.qprofiler2.summarise.ReadGroupSummary;
 import org.qcmg.qprofiler2.util.XmlUtils;
 
 
-public class BamSummaryReportTest {
+public class BamSummaryReportTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/bam/BamSummaryReportTest.java:27]: same

File: qprofiler2/test/org/qcmg/qprofiler2/bam/PositionSummaryTest.java [hunk 39]
```diff
@@ -9,13 +9,13 @@
 import org.qcmg.qprofiler2.summarise.PositionSummary;
 
 
-public class PositionSummaryTest {
+public class PositionSummaryTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/bam/PositionSummaryTest.java:12]: same

File: qprofiler2/test/org/qcmg/qprofiler2/bam/TagSummaryReportTest.java [hunk 40]
```diff
@@ -15,73 +15,73 @@
...
 import htsjdk.samtools.SAMTagUtil;
 
-public class TagSummaryReportTest {
+public class TagSummaryReportTest { 
 	@Rule
 	public  TemporaryFolder testFolder = new TemporaryFolder();	
 	
 	@Test
-	public void simpleTest() throws Exception{
+	public void simpleTest() throws Exception {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/bam/TagSummaryReportTest.java:30]: same

File: qprofiler2/test/org/qcmg/qprofiler2/cohort/CohortSummaryReportTest.java [hunk 41]
```diff
@@ -20,21 +20,21 @@
...
 
-public class CohortSummaryReportTest {
+public class CohortSummaryReportTest { 
 	
 	@Rule
 	public  TemporaryFolder testFolder = new TemporaryFolder();	
 	
 	private String sample = "https://test.sample";
-	private final static String[] category = new String[] {"FT","INF"};
+	private static final String[] category = new String[] { "FT","INF"};
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/cohort/CohortSummaryReportTest.java:29]: same

File: qprofiler2/test/org/qcmg/qprofiler2/summarise/CycleSummaryTest.java [hunk 42]
```diff
@@ -10,39 +10,39 @@
...
-import org.qcmg.qprofiler2.bam.BamSummarizer2;
-import org.qcmg.qprofiler2.bam.BamSummaryReport2;
+import org.qcmg.qprofiler2.bam.BamSummarizer;
+import org.qcmg.qprofiler2.bam.BamSummaryReport;
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
 
-public class CycleSummaryTest {
+public class CycleSummaryTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/summarise/CycleSummaryTest.java:19]: same

File: qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java [hunk 43]
```diff
@@ -34,40 +34,40 @@
 import htsjdk.samtools.SAMRecord;
 import htsjdk.samtools.SamReader;
 
-public class KmersSummaryTest {
+public class KmersSummaryTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java:37]: same

File: qprofiler2/test/org/qcmg/qprofiler2/summarise/PairSummaryTest.java [hunk 44]
```diff
@@ -17,56 +17,56 @@
...
-import org.qcmg.qprofiler2.bam.BamSummaryReport2;
+import org.qcmg.qprofiler2.bam.BamSummarizer;
+import org.qcmg.qprofiler2.bam.BamSummaryReport;
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
 import htsjdk.samtools.SAMRecord;
 
-public class PairSummaryTest {
+public class PairSummaryTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/summarise/PairSummaryTest.java:27]: same

File: qprofiler2/test/org/qcmg/qprofiler2/summarise/ReadGroupSummaryTest.java [hunk 45]
```diff
@@ -29,19 +29,19 @@
 import htsjdk.samtools.SAMRecord;
 import htsjdk.samtools.SamReader;
 
-public class ReadGroupSummaryTest {
+public class ReadGroupSummaryTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/summarise/ReadGroupSummaryTest.java:32]: same

File: qprofiler2/test/org/qcmg/qprofiler2/summarise/SampleSummaryTest.java [hunk 46]
```diff
@@ -20,21 +20,21 @@
 import org.qcmg.qprofiler2.util.XmlUtils;
 import org.w3c.dom.Element;
 
-public class SampleSummaryTest {
+public class SampleSummaryTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/summarise/SampleSummaryTest.java:23]: same

File: qprofiler2/test/org/qcmg/qprofiler2/util/CycleSummaryUtilsMDTest.java [hunk 47]
```diff
@@ -23,29 +23,29 @@
 import org.qcmg.qprofiler2.summarise.CycleSummaryTest;
 import org.qcmg.qprofiler2.util.CycleSummaryUtils;
 
-public class CycleSummaryUtilsMDTest {
+public class CycleSummaryUtilsMDTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/util/CycleSummaryUtilsMDTest.java:26]: same

File: qprofiler2/test/org/qcmg/qprofiler2/util/FlagUtilTest.java [hunk 48]
```diff
@@ -16,105 +16,106 @@
 import org.qcmg.qprofiler2.util.FlagUtil;
 import org.w3c.dom.Element;
 
-public class FlagUtilTest {
+public class FlagUtilTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/util/FlagUtilTest.java:19]: same

File: qprofiler2/test/org/qcmg/qprofiler2/util/SummaryReportUtilsTest.java [hunk 49]
```diff
@@ -10,58 +10,58 @@
 import org.qcmg.common.model.QCMGAtomicLongArray;
 import org.qcmg.qprofiler2.util.SummaryReportUtils;
 
-public class SummaryReportUtilsTest {
+public class SummaryReportUtilsTest {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/util/SummaryReportUtilsTest.java:13]: same

File: qprofiler2/test/org/qcmg/qprofiler2/vcf/VcfSummaryReportTest.java [hunk 50]
```diff
@@ -26,57 +26,57 @@
...
+public class VcfSummaryReportTest { 
 	
-	private final static String inputfile = "input.vcf";
-	private final static String[] category = new String[] {"FT","INF"};
+	private static final String inputfile = "input.vcf";
+	private static final String[] category = new String[] { "FT","INF"};
 	
 	@Test
-	public void HeaderTest() throws Exception{
+	public void HeaderTest() throws Exception {
```
Comment (delocalizer) [inline qprofiler2/test/org/qcmg/qprofiler2/vcf/VcfSummaryReportTest.java:35]: same

File: qprofiler2/src/org/qcmg/qprofiler2/summarise/SampleSummary.java (hunk 33 above)
Comment (ChristinaXu2017) [reply qprofiler2/src/org/qcmg/qprofiler2/summarise/SampleSummary.java:72]: I only fix issues on src, due to style check didn't check code under test.  No worries, I can fix it now.

File: qprofiler2/test/org/qcmg/qprofiler2/summarise/SampleSummaryTest.java (hunk 46 above)
Comment (ChristinaXu2017) [reply qprofiler2/test/org/qcmg/qprofiler2/summarise/SampleSummaryTest.java:19]: done

**Pull request merged**